build_debian.sh

```diff
@@ -221,7 +221,8 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     unzip                   \
     gdisk                   \
     sysfsutils              \
-    grub2-common
+    grub2-common            \
+    ipmitool
```

Comment:
can we move this into pmon docker? will it work?

Reply:
@jleveque

Reply:
Ipmitool is common utility.  It should put to "/usr/bin" for user application can access BMC.
If there is any concern on SONIC project,  please tell me how to add ipmitool cmd to pmon docker.  
I need to study and test on as7716-32XB  BMC project. Thanks.

Reply:
We study pmon docker and ipmitool use.  We think ipmitool is like i2c-tool that are common utility tool.  Ipmitool should put to /usr/bin or usr/sbin in SONiC rootfs, not put to any docker container. For BMC project, user only can access i2c from ipmitool. For non-MBC project, user can access i2c from i2c-tool.